vendor/github.com/docker/docker/pkg/jsonmessage/jsonmessage.go

```diff
@@ -48,10 +48,10 @@ func (p *JSONProgress) String() string {
 		timeLeftBox string
 	)
 
-	ws, err := term.GetWinsize(p.terminalFd)
-	if err == nil {
-		width = int(ws.Width)
-	}
+	//	ws, err := term.GetWinsize(p.terminalFd)
```

Comment:
We aren't doing any terminal i/o and this defaults to 200 for the width if it can't sniff out the terminal. Should be ok.